mapclassify/classifiers.py

```diff
@@ -182,7 +182,7 @@ def _get_table(mc, fmt="{:.2f}"):
     count_width = max(count_width, len("count"))
     interval_width = max(interval_width, len("interval"))
     header = "{:^{width}}".format("Interval", width=interval_width)
-    header += "   " + "{:>{width}}".format("Count", width=count_width)
+    header += " " * 3 + "{:>{width}}".format("Count", width=count_width)
     title = "{:<{width}}".format(mc.name, width=len(header))
```

Comment:
I probably could have explained that better. The extra space I mean are those added *after* `mc.name` in:
`"{:<{width}}".format(mc.name, width=len(header))`

For example:

```
BoxPlot
```
becomes
```
BoxPlot                 
```

Hard to see the formatting here, but clear in the screen shot below.

<img width="272" alt="Screenshot 2022-11-07 at 7 59 46 PM" src="https://user-images.githubusercontent.com/8590583/200448067-5c5d74fa-5e81-480a-9f33-74b96aea4bbe.png">

Are those extra spaces after the plot title needed for table alignment? CI passes when `title = mc.name` instead of the extra spaces for right justification, with output below:

```python
from libpysal import examples
import geopandas as gpd
from mapclassify import classify
link_to_data = examples.get_path('columbus.shp')
gdf = gpd.read_file(link_to_data)
x = gdf['HOVAL'].values
quantiles = classify(x, 'quantiles')
box_plot = classify(x, 'box_plot', hinge=2)
box_plot
```
current way (extra spaces for right justified)
```
BoxPlot               

   Interval      Count
----------------------
( -inf, -9.50] |     0
(-9.50, 25.70] |    13
(25.70, 33.50] |    12
(33.50, 43.30] |    12
(43.30, 78.50] |     9
(78.50, 96.40] |     3
```
proposed way (no extras spaces after title)
```
BoxPlot

   Interval      Count
----------------------
( -inf, -9.50] |     0
(-9.50, 25.70] |    13
(25.70, 33.50] |    12
(33.50, 43.30] |    12
(43.30, 78.50] |     9
(78.50, 96.40] |     3
```

Reply:
Ah, I see now what you are getting at.
Since the title is left justified, the trailing spaces can go as in the proposed approach.